spec/lib/tasks/date_import_field_spec.rb

```diff
@@ -34,13 +37,22 @@
     context "and we update the tenancycode field" do
       let(:field) { "tenancycode" }
 
-      it "updates the logs from the given XML file" do
+      it "updates the logs from the given XML file when the VCAP_SERVICES environment variable exists" do
         expect(Storage::S3Service).to receive(:new).with(paas_config_service, instance_name)
         expect(storage_service).to receive(:get_file_io).with("spec/fixtures/imports/logs")
         expect(Imports::LettingsLogsFieldImportService).to receive(:new).with(archive_service)
         expect(import_service).to receive(:update_field).with(field, "logs")
         task.invoke(field, fixture_path)
       end
+
```

Comment:
I didn't think it was worth adding a PaaS/Env test (like this one) for every context block in this file. I felt that testing it once was sufficient.